elixir/plan.sh

```diff
@@ -1,14 +1,15 @@
 pkg_origin=core
 pkg_name=elixir
-pkg_version=1.3.2
+pkg_version=1.3.4
+pkg_description="A dynamic, functional language designed for building scalable and maintainable applications. Elixir leverages the Erlang VM, known for running low-latency, distributed and fault-tolerant systems, while also being successfully used in web development and the embedded software domain."
+pkg_upstream_url=http://elixir-lang.org
 pkg_maintainer="Shane Sveller <shane@shanesveller.com>"
```

Comment:
As this is a core plan, this should also be updated to reflect the core maintainer metadata. See https://github.com/habitat-sh/core-plans/blob/master/CONTRIBUTING.md#package-metadata